blueocean-dashboard/src/main/js/util/logDisplayHelper.js

```diff
@@ -49,9 +49,10 @@ export const getNodesInformation = (nodes) => {
         /*
          * are we in a node that indicates that we have parallel nodes?
          */
-        if (item.edges && item.edges.length >= 2) {
-            parallelNodes = item.edges.map((itemParallel) => itemParallel.id);
+        if (item.edges) {
+            parallelNodes = item.edges.filter((edge) => edge.type === 'PARALLEL');
```

Comment:
Changing this from a `map` to a `filter` is going to change the structure of the array  (from an array of ID's to an array of nodes, I believe). Later below on line 57 there is an `parallelNodes.indexOf(item.id)` check that seems like it will fail 100% of the time if we aren't still calling `map` to project the ID's into the array. I am not really familiar with the code so maybe it doesn't matter.

Reply:
Good catch. Fixing it.